R/summaries.R

```diff
@@ -13,7 +13,10 @@
 #'   median or prevalence. Default is 'mean'.
 #'
 #' @param abund_values A \code{character} value to select an
-#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assayNames}}
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assayNames}} 
+#'   
+#' @param na.rm For \code{getTopTaxa} logical argument for calculation method 
+#'              specified to argument \code{method}. Default is TRUE. 
```

Comment:
These "methods" functions have also other possible arguments available, besides `na.rm`. 

We could allow the user to pass any of those available arguments through the "..." mechanism, I think we should allow this unless there are specific reason to not allow it. Then we could just remove the na.rm param description and state in "..." argument documentation (line 21 currently) that any other arguments available through \code{method} can be passed here, and cite the documentation of those functions for more details.

Reply:
@microsud

Reply:
If we add `...` then every time we may need to specify `na.rm` in the function. By using `na.rm=TRUE` as default IMO is more user friendly.  Other arguments are never used, at least I have never used it for this particular function which returns specific output.